github-fetcher/Makefile

```diff
@@ -0,0 +1,4 @@
+all: test
+.PHONY: test
```

Comment:
`test` is a [PHONY](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) target ...